options.go

```diff
@@ -23,9 +24,9 @@ type options struct {
 
 	*DumpOptions
 
-	LogLevel int
+	LogLevel Lever
 	// Logger *os.File
-	Logger atomic.Value
+	Logger Logger
```

Comment:
we should use atomic.Value to avoid concurrency read/write race.
```suggestion
	Logger atomic.Value // Logger
```

Reply:
atomic.Value 似乎不支持同接口不同实现的存储